demos/demos_databases_apis/microsoft/kusto/graphistry_ADX_kusto_demo.ipynb

```diff
@@ -0,0 +1,452 @@
+    "\n",
+    "### What’s New in Graphistry:\n",
+    "- **Native KQL Support**: You can now use KQL directly within Graphistry to run ADX queries and return dataframes.\n",
+    "- **Persistent Graph Queries**: Pass the name of your persistent graph and perform investigations in Graphistry's GPU-accelerated visual interface..\n",
+    "\n",
+    "\n",
+    "### Why It Matters:\n",
+    "With persistent graph semantics and Graphistry’s GPU-accelerated visual interface, analysts can now: Query, explore, and visualize ADX graph data with minimal friction.\n",
+    "\n",
+    "---\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a761a5b8-5a67-4d18-a72f-f04387aad089",
+   "metadata": {},
+   "source": [
+    "# Setup\n",
+    "\n",
+    "Install pygraphistry and the [Kusto python client](link), and get a free [Graphistry Hub GPU API key](https://hub.graphistry.com) or run your own [server](https://www.graphistry.com/get-started)\n",
+    "\n",
+    "\n",
+    "Install Graphistry and start your Kusto graph journey.\n",
+    "```bash\n",
+    "# Just Graphistry; bring your own Kusto install\n",
+    "pip install graphistry\n",
+    "\n",
+    "# Bundled Kusto install\n",
+    "pip install graphistry[kusto]\n",
+    "\n",
+    "```\n",
+    "---\n",
+    "Python:\n",
+    "```python\n",
+    "import graphistry\n",
+    "\n",
+    "graphistry.register()\n",
+    "graphistry.configure_kusto(cluster=KUSTO_CONF['cluster'], database=KUSTO_CONF['database'])\n",
+    "\n",
+    "df_5 = graphistry.kql(\"YourTable | take 5\")\n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b579c55-a691-49c3-a962-f88982722c00",
+   "metadata": {},
+   "source": [
+    "# Taking it for a spin\n",
+    "\n",
+    "To learn more about different authentication methods check out [API authentication to Graphistry servers](https://pygraphistry.readthedocs.io/en/0.35.2/server/register.html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e84ffb6f-baf5-46a4-bb4e-ea0ac6bf0731",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import graphistry"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46524eb4-2781-4996-a8f8-f594f8c2e28d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KUSTO_CONF = {\n",
+    "    \"cluster\": \"https://<clustername>.<region>.kusto.windows.net\",\n",
+    "    \"database\": \"YourDatabase\"\n",
+    "}\n",
+    "\n",
+    "GRAPHISTRY_CONF = {\n",
+    "    \"personal_key_id\": \"YOUR_KEY_ID\",\n",
+    "    \"personal_key_secret\": \"YOUR_SECRET\",\n",
+    "    \"server\": \"hub.graphistry.com\"\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "683b3d47-667f-4002-acda-cd1a6b56329c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "graphistry.register(api=3, **GRAPHISTRY_CONF)\n",
```

Comment:
Why do we need to call twice the register method? We already called it in the previous cells.